android-app-with-rn/android/app/build.gradle

```diff
@@ -11,7 +11,7 @@ datadogProperties.load(new FileInputStream(datadogPropertiesFile))
 
 android {
     namespace 'com.example.hybridapp'
-    compileSdk 32
+    compileSdk 33
```

Comment:
The latest one is 34. It is worth to update `targetSdk` below as well.

Also it makes sense to bump JVM target from 1.8 to 11 in the configuration below (SDK v2 anyway is shipped with JVM 11 bytecode).